### ISSUE TITLE
dbf won't open in GeoDa

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lixun910/geoda#204]
```
For some reason a dbf (and thus associated shapefile) won't open in GeoDa 
("error reading dbf) even though it opens without problem in ArcGIS. I deleted 
all columns with special formatting (dates, special characters), so not sure 
what the reason is. Will send you the shapefile by email. thx!
```

Original issue reported on code.google.com by `jkoschin...@gmail.com` on 23 Jul 2015 at 4:50

Attachments:
* [Screen Shot 2015-07-23 at 9.44.29 AM.png](https://storage.googleapis.com/google-code-attachments/geoda/issue-204/comment-0/Screen Shot 2015-07-23 at 9.44.29 AM.png)

**Comment by jkoschinsky** [comment]
switched to milestone 1.8 -- testing

**Comment by jkoschinsky** [comment]
test2 - ignore

**Comment by lixun910** [comment]
Using GDAL/OGR driver can solve the problem. 
Let's try to temporarily switch to GDAL/OGR on shapefile, and keep an eye on the performance and potential issues.

Fix will be in 1.7.51.

**Comment by lixun910** [comment]
reopen to verify

**Comment by jkoschinsky** [comment]
Xun, do we have a dbf that had this problem? I can't replicate with the ones I have since they all open fine.

**Comment by jkoschinsky** [comment]
found one: graz.shp (sent in separate email on Nov. 10, 2015)

**Comment by jkoschinsky** [comment]
![tix 204_change txt](https://cloud.githubusercontent.com/assets/13679743/11098407/c5209796-8872-11e5-885f-04a32ac857a8.png)

**Comment by jkoschinsky** [comment]
Verified fix in GeoDa 1.7.85 (OSX Yosemite, 10.10.5 and Win 7 Professional (64-bit)).
